lua/net.lua

```diff
@@ -13,6 +17,20 @@ function M.setup_block()
 	conf.num_rx_queues = 3
 	conf.num_tx_queues = 3
 
+	conf.front.num_ports = #front_ports
+	conf.front.name = "front"
+	conf.front.pci_addrs = ffi.new("const char *[" .. #front_ports .. "]")
```

Comment:
You'll need a C function to properly work here. The current code is wrong in a subtle way: the strings held in `front_ports` and `back_ports` are allocated by Lua, and Lua will release those string once it is done setting Gatekeeper up. The problem will come up when `conf.*.pci_addrs` is accessed later on.